tests/test_flow/test_persistence_fuzz.py

```diff
@@ -181,25 +208,25 @@ def add_entities(self, n_entities):
             all_names = self.model.entity_names()
 
             dep_names = []
+            changes_per_run = self._random_bool()
             for name in all_names:
                 if self._random_bool():
                     dep_names.append(name)
-            new_name = self.model.add_entity(dep_names)
+            new_name = self.model.add_entity(dep_names, changes_per_run)
 
             self.model.build_flow().get(new_name)
-            assert self.model.called_entity_names() == [new_name]
+            self.model.reset_called_entity_names()
```

Comment:
Is there any sensible assert we could use to replace what we used to have here?

Reply:
The assertion here would really be what is asserted between lines 331-337. I can move that into a separate method and call that assert here.

Reply:
The assertion here would really be what is asserted between lines 331-337. I can move that into a separate method and call that assert here.

Reply:
Sure